src/store/token-dashboard/sagas.ts

```diff
@@ -128,6 +132,19 @@ function* claim() {
   yield put(setModalState({ modalState: claimedState }))
 }
 
+const getSignableData = () => {
+  const vals = 'abcdefghijklmnopqrstuvwxyz123456789'
+  return vals.charAt(Math.floor(Math.random() * vals.length))
+}
+
+function* watchForDiscordCode() {
```

Comment:
shouldn't we do this one on dashboard page load rather than every time the dapp loads? signing is kinda expensive

Reply:
done